src/api/streaming/06-adaptive-streaming.md

```diff
@@ -21,7 +21,7 @@ video-subtitles:
     kind: captions
     label: Česky
     src: https://storage.googleapis.com/kino-assets/adaptive-streaming/cap-cz.vtt
```

Comment:
Shall we rename all vtt files as well for consistency or shall we keep `cz` in `srclang`?

```suggestion
    src: https://storage.googleapis.com/kino-assets/adaptive-streaming/cap-cs.vtt
```

Reply:
@beaufortfrancois Good point. Both the `srclang` attribute and the Google Cast media language code expect that code to be a [BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag), which means that Czech should be represented by `cs`. So I'd propose we change the `-cz` part of each subtitles name to `-cs` in the bucket and in our codebase.

@derekherman I'm not allowed to change filenames in the bucket, could you please do the changes and also change the subtitle URLs in `src/api/**/*.md` correspondingly?

Reply:
@dero This requires some coordination. How about we chat about it tonight in our sync and sort it out in realtime.